### PR TITLE
ENCD- 5125-add-factorbook

### PR DESCRIPTION
**Pull request by meenakshikagda**

### CODE REVIEW

File: src/encoded/schemas/namespaces.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "GEO": "http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=",
     "Caltech": "http://jumpgate.caltech.edu/library/",
     "FlyBase": "http://flybase.org/cgi-bin/quicksearch_solr.cgi?caller=quicksearch&tab=basic_tab&data_class=FBgn&species=Dmel&search_type=all&context=",
+    "FactorBook": "http://factorbook.beta.wenglab.org/experiment/",
     "WormBase": "http://www.wormbase.org/species/c_elegans/gene/",
```
Comment (gabdank) [inline src/encoded/schemas/namespaces.json]: would like to double-check with DAC is the URL not going to change to "non-beta" soon

File: src/encoded/static/components/dbxref.js [hunk 2]
```diff
@@ -107,6 +107,9 @@ export const dbxrefPrefixMap = {
     Cellosaurus: {
         pattern: 'https://web.expasy.org/cellosaurus/{0}',
     },
+    FactorBook: {
+        pattern: 'http://factorbook.beta.wenglab.org/experiment/{0}',
+    },
```
Comment (gabdank) [inline src/encoded/static/components/dbxref.js]: would like to double-check with DAC is the URL not going to change to "non-beta" soon

**Review by gabdank — CHANGES_REQUESTED** [review]
not approved until we hear back from DAC

File: src/encoded/static/components/dbxref.js (hunk 2 above)
Comment (hitz) [reply src/encoded/static/components/dbxref.js]: this page is SUPER slow

**Review by gabdank — APPROVED** [review]

**Pull request merged**